test/new-e2e/pkg/components/operator.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Looks like this component is never used, and has no client associated. You can skip it if it's not meant to receive an export from a pulumi output